google/earthenterprise
# Bundle Python for Builds and RPMs

Bundles python so that a we can rely on a specific version of python being available. Python for the build can now be built and installed via a shell script. Python is now built as part of the OpenGEE build, so that RPMs can be generated for distribution alongside the OpenGEE RPMs.

To test:
* Build and install opengee following the build instructions. Verify that OpenGEE works.
* Build and install the rpms, and verify that OpenGEE works.
